base_business_document_import/models/business_document_import.py

```diff
@@ -52,7 +52,7 @@ def _match_partner(
         base_phone_business_document_import
         """
         rpo = self.env['res.partner']
-        self._strip_cleanup_dict(partner_dict)
+        #self._strip_cleanup_dict(partner_dict)
```

Comment:
if this is bad for you, better override `_strip_cleanup_dict` to do what you need in your case